Cargo.toml

```diff
@@ -17,7 +17,7 @@ bundler = ["swc_bundler", "swc_ecma_transforms_optimization", "swc_graph_analyze
 cjs = ["utils", "visit"]
 codegen = ["swc_ecma_codegen", "swc_ecma_codegen_macros", "swc_macros_common"]
 compat = ["transforms", "swc_ecma_transforms_compat", "swc_trace_macro", "swc_config", "swc_config_macro"]
-dep_graph = ["swc_ecma_dep_graph"]
+dep_graph = ["visit"]
```

Comment:
Feel like "dep_graph" isn't a great name because it's not really a graph.

Reply:
dep_collect?

Reply:
I'll rename it in a future PR. Maybe dep_analysis